emma-language/src/main/scala/org/emmalanguage/compiler/lang/core/Core.scala

```diff
@@ -345,15 +348,15 @@ private[compiler] trait Core extends Common
     // -------------------------------------------------------------------------
 
     /** Delegates to [[CSE.transform()]]. */
-    lazy val cse: u.Tree => u.Tree =
+    lazy val cse: TreeTransform =
       CSE.transform(Map.empty, Map.empty)
 
     // -------------------------------------------------------------------------
     // Pickling API
     // -------------------------------------------------------------------------
 
     /** Delegates to [[Pickle.prettyPrint]]. */
-    val prettyPrint = unQualifyStatics andThen Pickle.prettyPrint
+    val prettyPrint = unQualifyStatics.transformation andThen Pickle.prettyPrint
```

Comment:
We can get rid of `_.transformation` projections if `TreeTransform` extends `u.Tree => u.Tree`. Is there a case against it?